packages/theme/css/palette/neutral-next.css

```diff
@@ -1,34 +1,36 @@
+/**
+ * Generated by `scripts/build-json.mjs`
+ * Do not edit directly, this file was auto-generated.
+ */
+
 .salt-theme.salt-theme-next[data-mode="light"] {
   --salt-palette-neutral: var(--salt-color-gray-500);
+  --salt-palette-neutral-action-active: var(--salt-color-gray-800);
+  --salt-palette-neutral-action-hover: var(--salt-color-gray-600);
   --salt-palette-neutral-disabled: var(--salt-color-gray-500-40a);
   --salt-palette-neutral-readonly: var(--salt-color-gray-500-10a);
   --salt-palette-neutral-strong: var(--salt-color-gray-600);
   --salt-palette-neutral-strong-disabled: var(--salt-color-gray-600-40a);
-  --salt-palette-neutral-stronger: var(--salt-color-gray-700);
-  --salt-palette-neutral-strongest: var(--salt-color-gray-800);
   --salt-palette-neutral-weak: var(--salt-color-gray-400);
   --salt-palette-neutral-weak-disabled: var(--salt-color-gray-400-40a);
+  --salt-palette-neutral-weak-readonly: var(--salt-color-gray-400-10a);
```

Comment:
Net new in Figma, code only has `--salt-palette-neutral-weaker-readonly`, neither used by characteristics?